src/main/java/net/countercraft/movecraft/combat/features/tracers/TNTTracers.java

```diff
@@ -38,9 +49,25 @@ public TNTTracers(@NotNull PlayerManager manager) {
     public static void load(@NotNull FileConfiguration config) {
         TracerRateTicks = config.getDouble("TracerRateTicks", 5.0);
         TracerMinDistanceSqrd = config.getLong("TracerMinDistance", 60);
+        TracerDelayTicks = config.getLong("TracerDelayTicks", 5);
+        TracerExplosionDelayTicks = config.getLong("TracerExplosionDelayTicks", 20);
         TracerMinDistanceSqrd *= TracerMinDistanceSqrd;
         TracerParticle = Particle.valueOf(config.getString("TracerParticles", "FIREWORKS_SPARK"));
         ExplosionParticle = Particle.valueOf(config.getString("ExplosionParticles", "VILLAGER_ANGRY"));
+        TracerBlock = Material.valueOf(config.getString("TracerBlock", "COBWEB"));
+        TracerExplosionBlock = Material.valueOf(config.getString("TracerExplosionBlock", "GLOWSTONE"));
```

Comment:
Why keep track of the material when we're only using it inside this function?

Reply:
fixed issue